README.md

```diff
@@ -277,62 +287,137 @@ signIn.renderEl(
     // screen letting them know that an email is on the way.
     if (res.status === 'FORGOT_PASSWORD_EMAIL_SENT') {
       // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
       return;
     }
 
     // The user has started the unlock account flow, and is on the confirmation
     // screen letting them know that an email is on the way.
     if (res.status === 'UNLOCK_ACCOUNT_EMAIL_SENT') {
       // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
+      return;
+    }
+
+    // The user has started the activation flow, and is on the confirmation
+    // screen letting them know that an email is on the way.
+    if (res.status === 'ACTIVATION_EMAIL_SENT') {
+      // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - username (string)
+      return;
+    }
+
+    // The user has completed the registration flow
+    if (res.status === 'REGISTRATION_COMPLETE') {
+      // Any followup action you want to take
+      // Properties available on `res`:
+      // - status (string)
+      // - activationToken (string)
       return;
     }
 
     // The user has successfully completed the authentication flow
     if (res.status === 'SUCCESS') {
 
-      // Handle success when the widget is not configured for OIDC
-
       if (res.type === 'SESSION_STEP_UP') {
         // Session step up response
         // If the widget is not configured for OIDC and the authentication type is SESSION_STEP_UP,
         // the response will contain user metadata and a stepUp object with the url of the resource
         // and a 'finish' function to navigate to that url
+        // Properties available on `res`:
+        // - status (string)
+        // - user (object)
+        // - type (string)
+        // - stepUp (object)
+        //   - url (string)
+        //   - finish (function)
         console.log(res.user);
         console.log('Target resource url: ' + res.stepUp.url);
         res.stepUp.finish();
         return;
-      } else {
-        // If the widget is not configured for OIDC, the response will contain
-        // user metadata and a sessionToken that can be converted to an Okta
-        // session cookie:
+      }
+
+      if (res.next) {
+        // If the response contains a redirect URL, the response object will contain a
+        // function that redirects the user when called.
+        // Properties available on `res`:
+        // - status (string)
+        // - user (object)
+        // - type (string)
+        // - next (function)
         console.log(res.user);
```

Comment:
`nit:` Just curious as to why we need this `console.log` in readme? Since it doesn't add any value, can this be cleaned up?

Also, if this is going to be cleaned up, perhaps we can clean up other places in readme as well since this PR is changing readme.

Reply:
Yeah, I was wondering about the `console.log` too. My best guess is that it was to help show what properties were available to the user in each scenario, but if we like the comment block documentation then I think removing the log statements as duplicative makes sense. WDYT?

Reply:
Imho, you're right, we don't need console.log. The updates you are doing indeed are helpful and are more elegant way of documentation. Hence `console.log` s can be removed. I'll let @haishengwu-okta and @aarongranick-okta chime in as well..

Reply:
I'll make this change based on no objections so far